XboxHelper.py

```diff
@@ -7,6 +7,9 @@
 import atexit
 import time
 
+# For general information on PFIFO, see
+# https://envytools.readthedocs.io/en/latest/hw/fifo/intro.html
```

Comment:
The naming by envytools / nouveau diverges from the nvidia names which were part of the Open-Source linux drivers and some reverse-engineered code.

Usually, the Xbox community sticks to the proprietary names due to the incompleteness of the nouveau names (although that sucks, because the proprietary names are hard to come by for most people. Those who have them usually whitelist those they are using on a case-by-case basis).

Reply:
I didn't look into specific registers (and don't have time for it), but I assume nv2a-trace already uses the XQEMU names?

I believe the XQEMU header should be using the nvidia names.

Reply:
> I didn't look into specific registers (and don't have time for it), but I assume nv2a-trace already uses the XQEMU names?

It does now, before this PR it was mostly using hex values directly and where it was using names they were not always aligned with the XQEMU conventions.
 
> I believe the XQEMU header should be using the nvidia names.

Very glad to hear it; I think it's also the easiest source of truth for the average contributor.